### ISSUE TITLE
Not an Issue but a doubt

### ISSUE THREAD

**Issue by PedroDrago** [refractalize/oil-git-status.nvim#5]
Is there a way trought configuration to disable the plugin error messages when git status failed?

Some way to disable this function or something like that:
```lua
if git_status_results.code ~= 0 then
	notify_system_error("Failed to load git status", git_status_results)
	return callback()
end
```

Thanks in advance!

**Comment by refractalize** [comment]
Hi @PedroDrago, thanks for the bug report!

This is fixed in 839a1a287f5eb3ce1b07b50323032398e63f7ffa

**Comment by PedroDrago** [comment]
Thanks!
